src/MetalPerformanceShaders/MPSImageLanczosScale.cs

```diff
@@ -1,19 +1,19 @@
+#if XAMCORE_2_0 || !MONOMAC
 // Copyright 2015 Xamarin Inc. All rights reserved.
 
-#if IOS
-
 using System;
 using System.Runtime.InteropServices;
 using XamCore.Foundation;
 using XamCore.Metal;
 using XamCore.ObjCRuntime;
 
 namespace XamCore.MetalPerformanceShaders {
-
+	[Mac (10, 13)]
 	public partial class MPSImageLanczosScale {
-
+		[Mac (10,13)]
```

Comment:
if the type is 10,13 then it's assumed all members are 10,13 (or later if decorated), i.e. there's no need for extra metadata on those elements.